tests/app/data_model/test_answer_store.py

```diff
@@ -267,3 +267,93 @@ def test_returns_ordered_map(self):
             self.assertGreater(instance, last_instance)
 
             last_instance = instance
+
+    def test_remove_answer(self):
+        answer_1 = Answer(
+            block_id="1",
+            answer_id="2",
+            answer_instance=1,
+            value=25,
+        )
+        answer_2 = Answer(
+            block_id="1",
+            answer_id="5",
+            answer_instance=1,
+            value=65,
+        )
+
+        self.store.add(answer_1)
+        self.store.add(answer_2)
+
+        expected_answers = {
+            "2_1": 25,
+            "5_1": 65,
+        }
+        self.assertEqual(self.store.map(), expected_answers)
+
+        self.store.remove(answer_2)
+        expected_answers = {
+            "2_1": 25,
+        }
+        self.assertEqual(self.store.map(), expected_answers)
+
+    def test_remove_answer_(self):
```

Comment:
The first assert is probably superfluous as I think there is already a separate test for the add. All I'm doing here is asserting that we did have 2 answers and after the remove we only have one.